nomos-services/blend/src/backends/libp2p.rs

```diff
@@ -149,6 +142,16 @@ where
                 panic!("Failed to listen on Blend network: {e:?}");
             });
 
+        // Dial the initial peers randomly selected
+        membership
+            .choose_remote_nodes(&mut rng, config.peering_degree)
```

Comment:
Where does this method comes from? If we added this to the MembershipHandler trait and I did not realize, this should not be there.

Reply:
This method is defined in the `Membership` struct. This struct is a different one from the membership in DA. Could you elaborate your question more?